generator/csv/fields.csv

```diff
@@ -909,7 +909,8 @@ SaveVehicleLocation,flash_current_level,f,Double,0x54,0.0,double
 SaveVehicleLocation,flash_time_left,f,Integer,0x55,0,int
 SaveVehicleLocation,unknown_65,f,Integer,0x65,0,?
 SaveVehicleLocation,original_move_route_index,f,Integer,0x66,0,Index of custom move route
-SaveVehicleLocation,unknown_6a,f,Integer,0x6A,0,?
+SaveVehicleLocation,remaining_ascent,f,Integer,0x6A,0,From 0 to 255 - In flying vehicles, remaining distance to ascend
+SaveVehicleLocation,remaining_descent,f,Integer,0x6B,0,From 0 to 255 - In flying vehicles, remaining distance to descend
```

Comment:
Lines 912 and 913 have extra "," in the comment column and breaks GitHub parser for web visualization. They could use ";" instead.